### ISSUE TITLE
[TalkBack] Explore By Touch event does not pass through SeekBar's AccessibiltyEvent handling methods

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#375]
```
What steps will reproduce the problem?
1. Create an app (attached) with a custom SeekBar and set ContentDescription to 
'Helpful text' in code (Activity.onCreate)
2. (Deliberately) set the ContentDescription to 'null' in 
onInitializeAccessibilityEvent, onPopulateAccessibilityEvent and 
dispatchAccessibilityEvent.
3. Enable Menu->Settings->Accessibility->TalkBack with 
TalkBack->Settings->Explore By Touch enabled.
4. Run the app and click on the custom SeekBar to trigger an Explore By Touch 
event.

What is the expected output? What do you see instead?
Expected result: The event should be handled as an AccessibilityEvent by the 
custom SeekBar. Hence, the text spoken should not include the original wording 
of the ContentDescription, as it is modified (removed) by the custom 
AccessibilityEvent handlers.
Actual result: TalkBack says 'Helpful text seek bar 0 per cent', incorrectly 
including the ContentDescription (and adding some extra words).

What version of the product are you using? On what operating system?
TalkBack: com.google.android.marvin.talkback-v3.5.0_r105.apk
Android: API 16 (same result on emulator and Samsung Galaxy Young)

Please provide any additional information below.

I discovered this issue while trying to add Accessibility support to my custom 
control which is based on SeekBar, but has quite different functionality. I am 
trying to remove the extra words being added, i.e. '...seek bar 0 per cent', 
which are meaningless in my implementation. Is this a bug, or there another way 
I can override it in code? Please advise.

I have attached a demo app, which illustrates the issue, and also raised a 
question on StackOverflow: 
http://stackoverflow.com/questions/25001069/android-how-to-eliminate-spoken-text
-from-accessibilityevents-when-extending-se
```

Original issue reported on code.google.com by `awayteam...@gmail.com` on 31 Jul 2014 at 9:53

Attachments:
- [HelloSeekBarAccessibility.tgz](https://storage.googleapis.com/google-code-attachments/eyes-free/issue-375/comment-0/HelloSeekBarAccessibility.tgz)
